ginga/examples/configs/plugin_SaveImage.cfg

```diff
@@ -0,0 +1,22 @@
+#
+# SaveImage plugin preferences file
+#
+# Place this in file under ~/.ginga with the name "plugin_SaveImage.cfg"
+
+# Default output parameters. Can also be changed in the GUI.
+output_directory = '.'
+output_suffix = 'ginga'
+
+# Clobber existing output files or not
+clobber = False
+
+# Only list modified images from the channel
+modified_only = True
```

Comment:
@ejeschke , to answer your question, if you set `modified_only = False`, it lists *all* images. Does this work for you?